integration-tests/cli/resource-deleter.it.js

```diff
@@ -0,0 +1,176 @@
+import fetch from 'node-fetch'
+import { createAuthMiddlewareForClientCredentialsFlow } from '@commercetools/sdk-middleware-auth'
+import { createClient } from '@commercetools/sdk-client'
+import { createRequestBuilder } from '@commercetools/api-request-builder'
+import { createHttpMiddleware } from '@commercetools/sdk-middleware-http'
+import ResourceDeleter from '@commercetools/resource-deleter'
+import { getCredentials } from '@commercetools/get-credentials'
+import { exec } from 'mz/child_process'
+import { version } from '@commercetools/resource-deleter/package.json'
+import {
+  carts,
+  categories,
+  channels,
+  customerGroups,
+  customers,
+  customObjects,
+  inventoryEntries,
+  payments,
+  productDiscounts,
+  products,
+  productTypes,
+  reviews,
+  shippingMethods,
+  taxCategories,
+  types,
+  zones,
+} from './helpers/resource-delete.data'
+import { createData, clearData } from './helpers/utils'
+
+let projectKey
+if (process.env.CI === 'true') projectKey = 'resource-deleter-int-test'
+else projectKey = process.env.npm_config_projectkey
+
+describe('Resource Deleter', () => {
+  let apiConfig
+  let resourceDeleter
+  let logger: {
+    error: () => {},
+    warn: () => {},
+    info: () => {},
+    debug: () => {},
+  }
+  const bin = './integration-tests/node_modules/.bin/resource-deleter'
+
+  function getResource(resource) {
+    const client = createClient({
+      middlewares: [
+        createAuthMiddlewareForClientCredentialsFlow({ ...apiConfig, fetch }),
+        createHttpMiddleware({ host: apiConfig.apiUrl, fetch }),
+      ],
+    })
+
+    const service = createRequestBuilder({
+      projectKey: apiConfig.projectKey,
+    })[resource]
+
+    const request = {
+      uri: service.build(),
+      method: 'GET',
+    }
+
+    return client.execute(request)
+  }
+
+  beforeAll(async () => {
+    // Get test credentials
+    const credentials = await getCredentials(projectKey)
+    apiConfig = {
+      host: 'https://auth.sphere.io',
```

Comment:
maybe this can be moved to some shared place so when we finally decide to leave sphere.io we can do it just once in constants file.